jsonslice_test.go

```diff
@@ -336,6 +336,37 @@ func Test_Expressions(t *testing.T) {
 	}
 }
 
+func Test_FuncNow(t *testing.T) {
+
+	tests := []struct {
+		name  string
+		Query string
+		isNil bool
```

Comment:
nitpick: would it be possible to specify the expected output in the test? it makes the test a bit more flexible/readable